libraries/AP_Mount/AP_Mount.h

```diff
@@ -28,6 +28,7 @@
 #include <AP_SerialManager/AP_SerialManager.h>
 #include <DataFlash/DataFlash.h>
 
+
```

Comment:
Do not add blank lines

Reply:
Sorry, should have checked before pushing. Fixed